test/basic.js

```diff
@@ -3,6 +3,7 @@ var spawn = require('child_process').spawn
 var signalExit = require('signal-exit')
 
 if (process.argv[2] === 'child') {
+  setTimeout(function(){}, 1000);
```

Comment:
i even though i can not pass test with coverage was a bug of `tap` 

thanks you for pointing out this!

👏